Python/Product/VSInterpreters/PackageManager/CPythonCondaPackageManagerProvider.cs

```diff
@@ -38,7 +38,8 @@ sealed class CPythonCondaPackageManagerProvider : IPackageManagerProvider {
 
         public IEnumerable<IPackageManager> GetPackageManagers(IPythonInterpreterFactory factory) {
             if (ExperimentalOptions.UseCondaPackageManager) {
-                if (!Directory.Exists(Path.Combine(factory.Configuration.PrefixPath, "conda-meta"))) {
+                if (string.IsNullOrEmpty(factory.Configuration.PrefixPath) ||
```

Comment:
You can extract `factory.Configuration.PrefixPath` into variable - it is used in several places below.

Reply:
sure